avocado/plugins/multiplex.py

```diff
@@ -59,6 +59,9 @@ def configure(self, parser):
         parser.add_argument('--list-variant-id', action='store_true',
                             default=False, help="Append variant id to list")
 
+        parser.add_argument('--get-count', action='store_true',
```

Comment:
So yep, this feature also looks useful.  It is most certainly based on your real world use cases, so it's a +1 on my book.